### PR TITLE
Automatically seed Trellis when spinning up platformdata service

### PR DESCRIPTION
**Pull request by mjgiarlo**
connects to LD4P/sinopia#196

Shows a pattern that the editor and server repos can use to automatically spin up Trellis with root, repo, and group containers + the root ACLs we have specified

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ $ npm test
 If you're going to be doing active development, you'll most likely want to spin up `docker-compose` services and run integration tests separately, since you will do this multiple times while getting the code right. If so, first spin up the integration environment—Trellis & its dependencies—in the background (via `-d`) using `docker-compose`:
 
 ```shell
-$ docker-compose up -d platform
+$ docker-compose up -d platformdata
```
Comment (ndushay) [inline README.md]: do you want to say anything about the diff b/t platformdata and platform?

Also, do you want to say anything about needing to set the env with the password to run the tests?

Comment (mjgiarlo) [reply README.md]: Changes coming!

Comment (mjgiarlo) [reply README.md]: Done.

**Comment by mjgiarlo** [comment]
@ndushay ready for another look. good eye! Thanks, the readme is now much better methinks.

File: Dockerfile [hunk 2]
```diff
@@ -4,4 +4,14 @@ WORKDIR /home/circleci
 
 COPY . .
 
+USER root
+
+# Allow circleci user to run babel-node executable
```
Comment (ndushay) [inline Dockerfile]: allow circle to run in circles!  🤣

Comment (mjgiarlo) [reply Dockerfile]: ha!

**Pull request merged**